### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [ ] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [ ] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 

If you are an Auth0 employee, after you submit this PR create and test a [Review App](https://github.com/auth0/docs#review-apps) to test your changes.
### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: articles/native-platforms/phonegap/01-login.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ description: This tutorial will show you how to use the Auth0 Phonegap SDK to ad
 ## Phonegap Tutorial
 
 <%= include('../../_includes/_package', {
```
Comment (mpaktiti) [inline articles/native-platforms/phonegap/01-login.md:14]: @mehreencs87 Please add the `githubUrl` line. 
[contributing guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects)

Also, can you please move the System Requirements panel at the top of the doc, before this include? Thanks!

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by mehreencs87** [comment]
@mpaktiti check the latest commit.

File: articles/native-platforms/phonegap/01-login.md [hunk 2]
```diff
@@ -4,24 +4,23 @@ description: This tutorial will show you how to use the Auth0 Phonegap SDK to ad
...
-**Otherwise, if you already have an existing application, please follow the steps below.**
-
 ::: panel-info System Requirements
 This tutorial and seed project have been tested with the following:
 
 * NodeJS 4.3
 * Phonegap 5.5
 :::
 
+<%= include('../../_includes/_package', {
```
Comment (mpaktiti) [inline articles/native-platforms/phonegap/01-login.md:14]: @mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
  Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this? Thanks.

**Comment by mehreencs87** [comment]
@mpaktiti check the latest commit.

**Pull request merged**